models/brats_mri_generative_diffusion/configs/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
-    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_generator_ldm_20230507.json",
-    "version": "1.0.9",
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20240725.json",
+    "version": "1.0.10",
```

Comment:
I guess here should be "1.1.0"?

Reply:
I was thinking we would change it to 1.1.0 when we switch to new model weights in the future? I'm fine with either way

Reply:
After discussed offline with @Nic-Ma, let's use `1.1.0` here since we removed generative dependency in this version.

Reply:
Yes, it should be a major update, thanks.

Reply:
Sure, will do. Thanks!

Reply:
Done, thanks